submit/workflow/processor.py

```diff
@@ -0,0 +1,82 @@
+"""Defines submission stages and workflows supported by this UI."""
```

Comment:
This is a new Class that handles where a submission is within a WorkflowDefinition.